arxiv/base/logging.py

```diff
@@ -16,26 +16,9 @@
 from datetime import datetime
 from pytz import timezone, utc
 
-from flask import request
-
 from arxiv.base.globals import get_application_config
 
 
-class RequestFormatter(logging.Formatter):
```

Comment:
Removing this since it is not used. Simplified to just a normal logger formatter.